ironfish-cli/src/commands/wallet/multisig/dkg/create.ts

```diff
@@ -117,16 +116,11 @@ export class DkgCreateCommand extends IronfishCommand {
       }
     }
 
-    const { name: participantName, identity } = ledger
-      ? await ui.retryStep(
-          () => {
-            Assert.isNotUndefined(ledger)
-            return this.getIdentityFromLedger(ledger, client, accountName)
-          },
-          this.logger,
-          true,
-        )
-      : await this.getParticipant(client, accountName)
```

Comment:
Same as above, can we delete this method?

Reply:
Also removed